bugbug/bug_features.py

```diff
@@ -896,3 +897,235 @@ def __call__(
+            "tsx",
+            "html",
+            "css",
+            "scss",
+            "rb",
+            "php",
+            "swift",
+            "kt",
+            "kts",
+            "go",
+            "rs",
+            "lua",
+            "sh",
+            "bat",
+            "ps1",
+            "pl",
+            "pm",
+            "r",
+            "m",
+            "mm",
+            "h",
+            "cs",
+            "vb",
+            "fs",
+            "fsharp",
+            "sln",
+            "vbproj",
+            "csproj",
+            "xaml",
+            "xml",
+            "json",
+            "yaml",
+            "yml",
+            "ini",
+            "cfg",
+            "conf",
+            "toml",
+            "md",
+            "txt",
+            "log",
+            "sql",
+            "db",
+            "sqlite",
+            "db3",
+            "csv",
+            "tsv",
+            "xlsx",
+            "xls",
+            "doc",
+            "docx",
+            "ppt",
+            "pptx",
+            "pdf",
+            "tex",
+            "bib",
+            "dvi",
+            "epub",
+            "mobi",
+            "azw3",
+            "rtf",
+            "odt",
+            "ods",
+            "odp",
+            "key",
+            "numbers",
+            "pages",
+            "gpx",
+            "kml",
+            "kmz",
+            "svg",
+            "ai",
+            "psd",
+            "xcf",
+            "png",
+            "jpg",
+            "jpeg",
+            "gif",
+            "bmp",
+            "tiff",
+            "ico",
+            "webp",
+            "mp3",
+            "wav",
+            "flac",
+            "ogg",
+            "m4a",
+            "wma",
+            "aac",
+            "mp4",
+            "mkv",
+            "avi",
+            "mov",
+            "flv",
+            "wmv",
+            "webm",
+            "3gp",
+            "m4v",
+            "mpg",
+            "mpeg",
+            "swf",
+            "fla",
+            "iso",
+            "img",
+            "dmg",
+            "zip",
+            "rar",
+            "tar",
+            "gz",
+            "bz2",
+            "xz",
+            "7z",
+            "s7z",
+            "rpm",
+            "deb",
+            "pkg",
+            "bin",
+            "exe",
+            "dll",
+            "so",
+            "o",
+            "a",
+            "lib",
+            "out",
+            "class",
+            "jar",
+            "war",
+            "ear",
+            "apk",
+            "ipa",
+            "plist",
+            "cfg",
+            "ini",
+            "conf",
+            "properties",
+            "yaml",
+            "yml",
+            "env",
+            "rc",
+            "htaccess",
+            "gitignore",
+            "dockerfile",
+            "makefile",
+            "gradle",
+            "pom",
+            "bat",
+            "cmd",
+            "vbs",
+        }
```

Comment:
same here

Reply:
Removed the hard-coded list, now using `pygments.lexers`: ca2a322b0cc35d8484082d9b777cad14de0ee23c